core/scripts/cli/sponsor/create.js

```diff
@@ -0,0 +1,42 @@
+const inquirer = require("inquirer");
+const getDefaultAccount = require("../wallet/getDefaultAccount");
+
+const create = async (web3, artifacts, emp) => {
+  const ExpandedERC20 = artifacts.require("ExpandedERC20");
+  const { toWei, fromWei, toBN } = web3.utils;
+
+  const sponsorAddress = await getDefaultAccount(web3);
+
+  const scalingFactor = toBN(toWei("1"));
+  // TODO: Understand why we need a .rawValue in one case but not the other.
+  const totalPositionCollateral = toBN((await emp.totalPositionCollateral()).rawValue.toString());
+  const totalTokensOutstanding = toBN((await emp.totalTokensOutstanding()).toString());
+  if (totalTokensOutstanding.isZero()) {
+    console.log("Error: can't create initial position with this tool");
```

Comment:
So I understand it that you don't allow a sponsor to create the first position (i.e. no other synthetic tokens outstanding)---why is that?

Reply:
We don't have a GCR in that case. The initial creation is a special case.

Reply:
Agreed we don't have a GCR. Do you currently have a logic branch where if its the sponsor's first position, you ignore GCR check?

Reply:
In the smart contract you mean? I'm assuming the smart contract handles that case correctly.

In this CLI, if the contract is empty, this check will fail. It doesn't matter if it's sponsor's first position, the check only applies to the globally first position.

Reply:
No I'm talking about the CLI: as you said if the contract is empty, then this check will fail and the `create()` will exit the function. From my understanding, that means you can't create new tokens if youre the globally first position--correct?

Reply:
That's correct.

Reply:
why not? just curious on why we block that

Reply:
It'd have to be a separate flow where we ask the user to put in the amount of collateral they'd want to put in. In this flow, the tool calculates the required amount of collateral.

Reply:
makes sense

Reply:
I think its worth adding a comment here that creating the global initial position would require a different workflow, which is why we're blocking it for now.

Reply:
Done.